### PR TITLE
Add new members to reviewers/approvers

### PR DESCRIPTION
**Pull request by dusk125**
https://bugzilla.redhat.com/show_bug.cgi?id=2082688

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,12 +1,15 @@
 approvers:
 - deads2k
-- hexfusion
 - hasbro17
+- dusk125
 reviewers:
 - deads2k
 - dusk125
 - hasbro17
 - hexfusion
```
Comment (hasbro17) [inline OWNERS]:
```suggestion
```

File: OWNERS [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
-- hexfusion
 - hasbro17
+- dusk125
 reviewers:
 - deads2k
 - dusk125
 - hasbro17
-- hexfusion
 - smarterclayton
 - wking
```
Comment (wking) [inline OWNERS:10]: Can you drop me, while you're touching this?  Looks like I was added way back in f7546eb638f47188dcee6b51fd62f7088cebd714, but this is not a repo I am actively watching, so any feedback from me on changes is going to be rare, best-effort sort of stuff.

**Pull request merged**